libsplinter/src/service/scabbard/factory.rs

```diff
@@ -30,26 +30,34 @@ use crate::signing::SignatureVerifierFactory;
 
 use super::{Scabbard, SERVICE_TYPE};
 
-const DEFAULT_DB_DIR: &str = "/var/lib/splinter";
-const DEFAULT_DB_SIZE: usize = 1028 * 1028 * 1028;
+const DEFAULT_STATE_DB_DIR: &str = "/var/lib/splinter";
+const DEFAULT_STATE_DB_SIZE: usize = 1028 * 1028 * 1028;
```

Comment:
Why are these 1028? (I realize this predates this commit)

Reply:
Just the precedent set by Sawtooth.

Reply:
Uhhh yeah that should actually be 1024. No idea why it's 1028.

Reply:
Fixed